playbooks/common/openshift-master/config.yml

```diff
@@ -17,6 +17,13 @@
     - node
     - .config_managed
 
+  - fail:
+      msg: "Both openshift_master_session_auth_secrets and openshift_master_session_encryption_secrets must be provided if either variable is set"
+    when: (openshift_master_session_auth_secrets is defined and openshift_master_session_encryption_secrets is not defined) or (openshift_master_session_encryption_secrets is defined and openshift_master_session_auth_secrets is not defined)
+  - fail:
+      msg: "openshift_master_session_auth_secrets and openshift_master_encryption_secrets must be equal length"
+    when: (openshift_master_session_auth_secrets is defined and openshift_master_session_encryption_secrets is defined) and (openshift_master_session_auth_secrets | length != openshift_master_session_encryption_secrets | length)
```

Comment:
I've been wondering if we need a "check mode" for variables...  maybe we should get better about tagging, and then we could tell someone to run the playbooks setting the "check_mode" tag to verify an inventory before attempting to configure it... We could then really fail fast for missing required settings from the installer.